WalletWasabi.Fluent/Views/Wallets/Send/TransactionPreviewView.axaml

```diff
@@ -27,6 +27,13 @@
                  ToolTip.Tip="Your privacy is being protected with payjoin."
                  IsVisible="{Binding CurrentTransactionSummary.IsPayJoin}" />
 
+          <Button Command="{Binding UndoCommand}" Classes="plain" Padding="10 0 10 10"
+                  ToolTip.Tip="Undo last modification" IsVisible="{Binding CanUndo}">
+            <Viewbox Height="20">
+              <PathIcon Data="{StaticResource undo_icon_regular}" Foreground="{DynamicResource SystemAccentColor}" />
+            </Viewbox>
+          </Button>
+
```

Comment:
The icon is too thick compared to any other. Can you make it a bit thinner?
![image](https://user-images.githubusercontent.com/16364053/157649947-0ff17090-11d0-40bb-814b-b92dcb4b830d.png)

Reply:
@soosr fixed